lib/osc.py

```diff
@@ -2882,13 +2919,14 @@ def uploadSslKeypairImage(self, imgPath=None):
         self._output.log_debug("uploadSslKeypairImage -- Image Path:  \"%s\"" %(imgPath))
         rstUrl = "/api/server/v1/serverManagement/internalkeypair"
         if imgPath:
+            imgName = os.path.basename(imgPath)
             if not os.path.exists(imgPath):
                 raise Exception("uploadSslKeypairImage -- Error: Img File \"%s\" Not Found" %(imgPath))
             pass
             if not len(imgPath):
                 raise Exception("uploadSslKeypairImage -- Error: Img File \"%s\" is empty" %(imgPath))
             pass
-
+            #rstUrl = (ssl_upload_url + r"/" + imgName)
```

Comment:
remove this line

Reply:
All changes done by rebase, edit, add, commit, fetch.